python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -88,6 +93,8 @@ class Output(Generic[T]):
             list, and one of the data classes returned by a MetadataValue static method.
         data_version (Optional[DataVersion]): (Experimental) A data version to manually set
             for the asset.
+        tags (Optional[Mapping[str, str]]): (Experimental) Tags that will be attached to the asset
```

Comment:
Should this be named something like `asset_event_tags`?

Reply:
At a minimum, I think it is important for the docstring to say "Tags that will be attached to the asset materialization"

My general feeling is that `tags` is sufficient.